crates/core/tedge_mapper/src/c8y/converter.rs

```diff
@@ -789,4 +802,14 @@ mod tests {
         // blocking and that you probably removed a tokio::spawn handle (;
         assert_eq!(now.elapsed().as_secs(), 0);
     }
+
+    #[test_case("cds50223434,uninstall-test"; "valid template")]
+    #[test_case("5000000000000000000000000000000000000000000000000,uninstall-test"; "long valid template")]
+    #[test_case(""; "empty payload")]
+    fn extract_smartrest_tempate(payload: &str) {
+        match super::get_smartrest_template(payload) {
+            Ok(_) => assert!(true),
```

Comment:
It would be good to check that smartrest template is the expected one ;-)

Reply:
added a check